example-library/pom.xml

```diff
@@ -27,11 +27,25 @@
     <artifactId>example-library</artifactId>
 
     <dependencies>
+        <dependency>
+            <groupId>com.google.protobuf</groupId>
+            <artifactId>protobuf-java</artifactId>
+            <version>${protobuf.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-netflix-eureka-client</artifactId>
+        </dependency>
```

Comment:
Needs justfication

Reply:
These are likely to have been left over from previous example work. Removed.